docs/docs/addons-test-utils.md

```diff
@@ -252,32 +252,5 @@ Same as [`scryRenderedComponentsWithType()`](#scryrenderedcomponentswithtype) bu
 
 ## Shallow Rendering
```

Comment:
Can we move this section right after Overview? It's what most people are looking for when testing React.